eks-without-vpc/subnets.tf

```diff
@@ -0,0 +1,35 @@
+#
+#  * Subnets
+#  * Route Table
+#
+
+resource "aws_subnet" "aws-eks" {
+  count = 2
+
+  availability_zone = "${data.aws_availability_zones.available.names[count.index]}"
+  cidr_block        = "${var.vpc-cidr-prefix}.${count.index + var.subnet-3rd-octet}.0/24"
+  vpc_id            = "${var.vpc-id}"
+
+  tags = "${
+    map(
+     "Name", "reltio-${var.cluster-name}-node",
```

Comment:
Also this tag name shouldn't include `reltio`

Reply:
changed